### PR TITLE
add a list of delicious teas

### PR DESCRIPTION
**Pull request by coleww**
culled from wikipedia :tea:

### CODE REVIEW

File: data/foods/tea.json [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+    "Mengding Ganlu",
+    "Micromeria chionistrae",
+    "Micromeria douglasii",
+    "Mushroom",
+    "Nepali",
+    "Nilgiri",
+    "Noon Chai",
+    "Oksusucha",
+    "Oolong",
+    "Oolong",
```
Comment (hugovk) [inline data/foods/tea.json]: Duplicate Oolongs.

File: data/foods/tea.json [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    "Tagetes lucida",
+    "Taiping houkui",
+    "Tamaryokucha",
+    "Tereré",
+    "Thai",
+    "Tibeti",
+    "Tieguanyin",
+    "Tieluohan",
+    "Tsheringma",
+    "Tulsi or Holy Basil",
```
Comment (hugovk) [inline data/foods/tea.json]: Perhaps split into two entries?

File: data/foods/tea.json [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+    "Nepali",
+    "Nilgiri",
+    "Noon Chai",
+    "Oksusucha",
+    "Oolong",
+    "Oolong",
+    "Peace Iced",
+    "Peace",
+    "Peppermint",
+    "Peppermint",
```
Comment (hugovk) [inline data/foods/tea.json]: Duplicate.

File: data/foods/tea.json [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    "Pouchong",
+    "Pu-erh",
+    "Pu-erh",
```
Comment (hugovk) [inline data/foods/tea.json]: Duplicate.

**Comment by hugovk** [comment]
Generally I'd encourage things to be in sentence case, that is: only cap things like proper nouns, so they can be slotted in anywhere in a sentence without looking odd. For example:

> Would you like a nice hot cup of `peppermint` tea? Or perhaps `Thai` tea?

Instead of:

> Would you like a nice hot cup of `Peppermint` tea? Or perhaps `Thai` tea?

If I want them all lowercase, or want them in Initial Case, or ALL CAPS, or want to cap the first word at the start of a sentence, that's easy with post-processing. But trying to guess correct caps after the fact is impossible.

**Comment by coleww** [comment]
@hugovk Sorry about all the dupes, thought I had cleaned it up better :dash: 

I went through and fixed the casing as best i could. Learned a lot about teas from googling every word on the list that I thought might be a proper/place-noun.

I'll keep that in mind next time i make a list from wiki data that I shouldn't just accept their capitalized article titles :<

**Pull request merged**